docs/Home.md

```diff
@@ -1,18 +1,71 @@
+
+This introduction is designed to provide you with an overview on Inductiva API, 
+you will learn:
+
+* [What is the Inductiva API?]()
+* [What are Inductiva API's core functionalities?]()
+* [What can you do?]()
+
+## What is the Inductiva API?
+
+Inductiva API is a Python package that enhances the usability and scalability of 
+[free and open-source physical simulation software packages](), which are pivotal 
+in advancing science and engineering. These software packages, while powerful, often 
+present significant challenges in terms of installation, operation, and particularly, 
+scaling for large-scale computations. This complexity can restrict the potential 
+of engineers and scientists to fully exploit these tools.
+
+Inductiva addresses these challenges by providing on-demand scaling capabilities 
+to high-performance systems, making it simpler for users to deploy these simulators 
+at scale without the necessity of specialized systems engineering or high-performance 
+computing (HPC) team support. 
+
+In essence, **Inductiva's focus lies in computation**: making advanced [computational resources]()
+user-friendly and within reach, opening up new possibilities in the field of 
+simulation-based research and analysis.
+
+## What are Inductiva API's core funtionalities?
+Inductiva designed the core funtionalities of its API tool to facilitate 
+various tasks in a computational environment with minimal coding effort. It will 
+allow you to:
+* easily dispatch 
+computational tasks,
+* manage system resources,
+* and run detailed simulations.
+
+You will interact with these cloud-based simulations through a Python client on 
+your local computer. This interface enables you to select, manage, and scale your 
+simulations in the cloud, leveraging the scalability and power of Inductiva's 
+[computational resources](), while bypassing the limitations of your local hardware.
+
+This API offers a range of benefits for diverse simulation needs:
+
+|  	|  	|
+|---	|---	|
+| Unified Platform 	| _It unifies multiple simulation packages on a single platform, covering various <br>multi-physics aspects that can encompass domains like fluid dynamics, molecular dynamics, <br>plasmas, and structural mechanics. This integration allows for a streamlined, user-friendly <br>access point, eliminating the need to manage separate software for each type of simulation._ 	|
+| Hassle-Free Setup 	| _It eliminates the need for installing and managing <br>complex simulation software and corresponding dependencies_ 	|
+| Python-Powered Flexibility 	| _It empowers you to write simple <br>Python scripts that blend seamlessly with your existing codebase and machine learning <br>frameworks, unlocking a world of customization and control._ 	|
+| Smart Hardware Optimization 	| _It automatically tunes hardware configurations – <br>whether you need CPU or GPU, decides the right number of cores, RAM, etc., specific <br>to each simulation type._ 	|
```

Comment:
I think it's more general than that, and in fact, this seems it is going one step too far.
I like talking about optimization but:

1) just the orchestration itself -- even if it is not optimized -- is a great feature. Setting up an MPI cluster is a huge thing. But also other things that we already have like auto-scaling of resources, and automatic termination of resources are super super important. I would prefer to talk more about this @hpenedones @sergiomsantos ?

2) We also need to make a reference to the cost and cost visibility @sergiomsantos ?